src/together/cli/api/finetune.py

```diff
@@ -99,6 +99,9 @@ def create(
 
     click.echo(json.dumps(response.model_dump(exclude_none=True), indent=4))
 
+    # TODO: Remove it after the 21st of August
+    log_warn("The default value of batch size has been changed from 32 to 16 since together version >= 1.2.6")
```

Comment:
I added it here because a formated FT JSON is quite big, and the warning can be easily missed by a user.